integration-tests/src/tests/client/features/orphan_chunk_state_witness.rs

```diff
@@ -232,55 +231,6 @@ fn test_orphan_witness_valid() {
         .unwrap();
 }
 
-#[test]
-fn test_orphan_witness_bad_signature() {
```

Comment:
These tests test signature which we are removing from state witness and pushing functionality to parts.